### PR TITLE
[core] Cleanup issues body

### LINKED ISSUE #10340: [core] Improve GitHub template
As of today, creating a new issue leads to an annoying doubleheader, it takes vertical space that readers can skip.

Screenshot 2023-09-14 at 19 47 59

What if we replaced this for a single header?

See how the lighthouse feels better: https://github.com/GoogleChrome/lighthouse/issues/15450.

Screenshot 2023-09-14 at 19 49 10

I have seen Toolpad (@Janpot can't find which issue again) complain about this as well, so feels like we might  be able to improve this for the whole MUI org.

### PR DESCRIPTION
**Pull request by romgrk**
Closes #10340

Remove the checkboxes from new issues' body. I have hardcoded the strings, seems good enough.

Result: https://github.com/romgrk/mui-test-issue-cleanup/issues/3

### CODE REVIEW

File: .github/workflows/issue-cleanup.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        with:
+          script: |
+            const issue = await github.rest.issues.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+            });
+
+            const text =
+              '### Duplicates\r\n\r\n- [X] I have searched the existing issues\r\n\r\n### Latest version\r\n\r\n- [X] I have tested the latest version\r\n\r\n';
```
Comment (romgrk) [inline .github/workflows/issue-cleanup.yaml:26]: Github seems to store newlines using windows conventions.

**Comment by Janpot** [comment]
> I have hardcoded the strings, seems good enough.

Would it work if we add

```md
  - type: markdown
    attributes:
      value: <!-- delete:start -->

  # checkboxes ...

  - type: markdown
    attributes:
      value: <!-- delete:end -->
```

And postprocess them with

```tsx
const body = issue.data.body
  .replaceAll('<!-- delete:start -->', '<!--')
  .replaceAll('<!-- delete:end -->', '-->')
```

Doesn't seem much more complicated, it doesn't have to be kept in sync with the template strings, and directly reusable in other templates.

**Review by Janpot — APPROVED** [review]
Works for me. Commented with an alternative implementation. But fine to merge as is as well.

**Comment by romgrk** [comment]
That's a neat idea, but we often edit issues main comment along the way, and leaving those comments in there would still be a bit annoying. But it's trivial to update in the future so I'll merge as is for now.

**Comment by Janpot** [comment]
> That's a neat idea, but we often edit issues main comment along the way, and leaving those comments in there would still be a bit annoying.

```tsx
const body = issue.data.body.replaceAll(/<!-- delete:start -->.*?<!-- delete:end -->/g, '')
```

**Pull request merged**